README.md

```diff
@@ -13,6 +13,12 @@
 	# or
 	yarn
 	```
+- Add environment variables in `.env.local`
+	```
+		NEXT_PUBLIC_API_BASE_URL=<base-url-of-deployed/local-api>
```

Comment:
@akshatbhargava123 We also need to inform the user to restart the server if they make changes to the `.env.local` file

Reply:
The streamlined flow already shows that user need to edit `.env.local` before running the development server and I don't feel a need for explicit mention here, do you still think it's worth mentioning?